planner/core/testdata/integration_serial_suite_in.json

```diff
@@ -194,7 +194,7 @@
       "desc format = 'brief' select * from t join ( select count(*), id from t group by id) as A on A.id = t.id",
       "desc format = 'brief' select * from t join ( select /*+ hash_agg()*/  count(*) as a from t) as A on A.a = t.id",
       "desc format = 'brief' select avg(value) as b,id from t group by id",
-      "desc format = 'brief' select /*+hash_agg()*/ sum(b),id from (select avg(value) as b, id from t group by id)A",
+      "desc format = 'brief' select /*+hash_agg()*/ sum(b) from (select avg(value) as b, id from t group by id)A",
```

Comment:
Tested on MySQL 8.0.23, this usage of `id` is incompatible with sql_mode=only_full_group_by, so I removed it.

Reply:
You mean that if we add `id` in the select fields, MySQL 8.0.23 would report `only_full_group_by` error?

Reply:
Yes.

Reply:
Sure, it's very hard to be fully compatible with MySQL in our current impl. :joy: